swarm/src/connection/pool.rs

```diff
@@ -237,6 +240,9 @@ where
         /// Addresses are dialed in parallel. Contains the addresses and errors
         /// of dial attempts that failed before the one successful dial.
         concurrent_dial_errors: Option<Vec<(Multiaddr, TransportError<TTrans::Error>)>>,
+        /// How long it took to establish this connection
```

Comment:
```suggestion
        /// How long it took to establish this connection.
```

Reply:
My childhood grammar teacher would be upset with me for using sentence punctuation (.) with out a main verb, but I doubt she'll see this 😊